pkg/coredns/coredns.go

```diff
@@ -100,3 +103,37 @@ func GetUserID() int64 {
 
 	return int64(uid)
 }
+
+func DeleteCoreDNSComponents(ctx context.Context, client *kubernetes.Clientset, namespace string) error {
+	labelSelector := labels.SelectorFromSet(labels.Set{"k8s-app": "vcluster-kube-dns"}).String()
```

Comment:
We're using `"k8s-app": "vcluster-kube-dns"` in multiple places and this may become a problem when we want to change it. Could you make it a constant and store in the `pkg/constants/coredns.go` and replace all occurrences of it in the code? Thanks